PROJECT

```diff
@@ -213,4 +213,43 @@ resources:
     defaulting: true
     validation: true
     webhookVersion: v1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: redhat.io
+  group: redhatcop
+  kind: QuaySecretEngineConfig
+  path: github.com/redhat-cop/vault-config-operator/api/v1alpha1
+  version: v1alpha1
+  webhooks:
+    defaulting: true
+    validation: true
+    webhookVersion: v1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: redhat.io
+  group: redhatcop
+  kind: QuaySecretEngineRole
+  path: github.com/redhat-cop/vault-config-operator/api/v1alpha1
+  version: v1alpha1
+  webhooks:
+    defaulting: true
+    validation: true
+    webhookVersion: v1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: redhat.io
+  group: redhatcop
+  kind: QuaySecretEngineStaticRole
```

Comment:
what is a static role? Can you explain this concept, other engines don't need it.

Reply:
This secrets engine is modeled in a similar way as the Database secrets engine. You can read [here](https://www.vaultproject.io/docs/secrets/databases#static-roles) on the concepts of Static Roles

Reply:
ok, now I see, but does it need to be a different API. Could it just be a flag?

Reply:
Right now it could. however, the next feature that will be added in the plugin is the ability to define a rotation period. Once again, could be added to a single Role CR, but could make things even less clear

Reply:
so we keep this as a separate API